client/src/app/site/motions/services/motion-repository.service.ts

```diff
@@ -66,14 +67,32 @@ export class MotionRepositoryService extends BaseRepository<ViewMotion, Motion>
      */
     public saveMotion(update: any, viewMotion?: ViewMotion): Observable<any> {
         let updateMotion: Motion;
-
         if (viewMotion) {
             // implies that an existing motion was updated
             updateMotion = viewMotion.motion;
         } else {
             // implies that a new motion was created
             updateMotion = new Motion();
         }
+        // submitters: User[] -> submitter: MotionSubmitter[]
+        const submitterArr = update.submitters as User[];
+        const newSubmitterArr: MotionSubmitter[] = [];
+        if (submitterArr.length > 0) {
+            submitterArr.forEach(user_submitter => {
+                const mot_submitter = new MotionSubmitter(user_submitter, updateMotion.id, 1);
```

Comment:
weight should accumulate, so four submitters have the weight: `1, 2, 3, 4` instead of just `1, 1, 1, 1`

```
submitterArr.forEach(user_submitter, index) => {
    const mot_submitter = new MotionSubmitter(user_submitter, updateMotion.id, index+1);
});
```
(not tested)